src/test/java/uk/gov/companieshouse/itemhandler/XxxxApiApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.companieshouse.itemhandler;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class XxxxApiApplicationTests {
+
```

Comment:
Strange test class name `XxxxApiApplicationTests`.

Reply:
This is a remnant from the template. I have fixed this now. Thanks